lib/ansible/modules/packaging/os/pacman.py

```diff
@@ -138,32 +138,40 @@ def get_version(pacman_output):
     return None
 
 
-def query_package(module, pacman_path, name, state="present"):
-    """Query the package status in both the local system and the repository. Returns a boolean to indicate if the package is installed, a second
-    boolean to indicate if the package is up-to-date and a third boolean to indicate whether online information were available
+def query_package_installed(module, pacman_path, name):
```

Comment:
What if we implement query_package_local and query_package_remote instead ?
(replacing query_package_installed and query_package_latest)
What do you think ? (just a suggestion)